model/src/test/kotlin/utils/FindingsMatcherTest.kt

```diff
@@ -37,8 +37,11 @@ import kotlin.random.Random
 
 private fun Collection<LicenseFindings>.getFindings(license: String) = single { it.license == license }
 
+private const val ROOT_LICENSE_FILE_1 = "a/LICENSE"
+private const val ROOT_LICENSE_FILE_2 = "b/LICENSE"
```

Comment:
I'd prefer the names `NESTED_LICENSE_A` and `NESTED_LICENSE_B` as these are not root licenses in the sense of being in the root directory.

Reply:
Done.